libs/zopengl/src/wrapper.zig

```diff
@@ -41,40 +41,259 @@ pub const Capability = enum(Enum) {
+    samples = SAMPLES,
+    sample_buffers = SAMPLE_BUFFERS,
+    sample_coverage_invert = SAMPLE_COVERAGE_INVERT,
+    sample_coverage_value = SAMPLE_COVERAGE_VALUE,
+    num_compressed_texture_formats = NUM_COMPRESSED_TEXTURE_FORMATS,
+    max_cube_map_texture_size = MAX_CUBE_MAP_TEXTURE_SIZE,
+    //---------------------------------------------------------------------------------------------
+    // OpenGL 1.4 (Core Profile)
+    //---------------------------------------------------------------------------------------------
+    blend_color = BLEND_COLOR,
+    blend_src_rgb = BLEND_SRC_RGB,
+    blend_src_alpha = BLEND_SRC_ALPHA,
+    blend_dst_rgb = BLEND_DST_RGB,
+    blend_dst_alpha = BLEND_DST_ALPHA,
+    point_fade_threshold_size = POINT_FADE_THRESHOLD_SIZE,
+    max_texture_lod_bias = MAX_TEXTURE_LOD_BIAS,
+    //---------------------------------------------------------------------------------------------
+    // OpenGL 1.5 (Core Profile)
+    //---------------------------------------------------------------------------------------------
+    array_buffer_binding = ARRAY_BUFFER_BINDING,
+    element_array_buffer_binding = ELEMENT_ARRAY_BUFFER_BINDING,
+    //---------------------------------------------------------------------------------------------
+    // OpenGL 2.0 (Core Profile)
+    //---------------------------------------------------------------------------------------------
+    current_program = CURRENT_PROGRAM,
+    blend_equation_rgb = BLEND_EQUATION_RGB,
+    blend_equation_alpha = BLEND_EQUATION_ALPHA,
+    stencil_back_fail = STENCIL_BACK_FAIL,
+    stencil_back_func = STENCIL_BACK_FUNC,
+    stencil_back_pass_depth_fail = STENCIL_BACK_PASS_DEPTH_FAIL,
+    stencil_back_pass_depth_pass = STENCIL_BACK_PASS_DEPTH_PASS,
+    stencil_back_ref = STENCIL_BACK_REF,
+    stencil_back_value_mask = STENCIL_BACK_VALUE_MASK,
+    stencil_back_writemask = STENCIL_BACK_WRITEMASK,
+    draw_buffer0 = DRAW_BUFFER0,
+    draw_buffer1 = DRAW_BUFFER1,
+    draw_buffer2 = DRAW_BUFFER2,
+    draw_buffer3 = DRAW_BUFFER3,
+    draw_buffer4 = DRAW_BUFFER4,
+    draw_buffer5 = DRAW_BUFFER5,
+    draw_buffer6 = DRAW_BUFFER6,
+    draw_buffer7 = DRAW_BUFFER7,
+    draw_buffer8 = DRAW_BUFFER8,
+    draw_buffer9 = DRAW_BUFFER9,
+    draw_buffer10 = DRAW_BUFFER10,
+    draw_buffer11 = DRAW_BUFFER11,
+    draw_buffer12 = DRAW_BUFFER12,
+    draw_buffer13 = DRAW_BUFFER13,
+    draw_buffer14 = DRAW_BUFFER14,
+    draw_buffer15 = DRAW_BUFFER15,
+    draw_framebuffer_binding = DRAW_FRAMEBUFFER_BINDING,
+    fragment_shader_derivative_hint = FRAGMENT_SHADER_DERIVATIVE_HINT,
+    max_combined_texture_image_units = MAX_COMBINED_TEXTURE_IMAGE_UNITS,
+    max_draw_buffers = MAX_DRAW_BUFFERS,
+    max_fragment_uniform_components = MAX_FRAGMENT_UNIFORM_COMPONENTS,
+    max_texture_image_units = MAX_TEXTURE_IMAGE_UNITS,
+    //max_varying_floats = MAX_VARYING_FLOATS, // NOTE: MAX_VARYING_FLOATS is equal to MAX_VARYING_COMPONENTS
```

Comment:
urgh!

Reply:
`MAX_VARYING_FLOATS` is an old name from pre OpenGL 3.0 - we can ignore it.